.travis.yml

```diff
@@ -125,10 +136,9 @@ install:
         sqlalchemy
         mock
         nose
-        gdal
```

Comment:
gdal is used in io.shapefile tests.. i suppose it was making problems with Py 3.6? maybe we can install it in at least a few Travis runs?